extensions/mssql/test/unit/schemaCompareWebViewController.test.ts

```diff
@@ -564,7 +564,7 @@ suite("SchemaCompareWebViewController Tests", () => {
         expect(generateScriptStub, "generateScript should be called once").to.have.been.calledOnce;
 
         expect(
-            generateScriptStub.firstCall.args,
+            generateScriptStub.firstCall.args.slice(0, 4),
```

Comment:
Funky.  Does `expect(publishProjectChangesStub).to.have.been.calledWith(xyz)` not work here?